src/api/libcellml/generator.h

```diff
@@ -107,6 +105,28 @@ class LIBCELLML_EXPORT Generator
      */
     std::string implementationCode() const;
 
+    /**
+     * @brief Get the equation code for the given @ref AnalyserEquationAst.
+     *
+     * Return the equation code for the given @ref AnalyserEquationAst using @ref GeneratorProfile, if present, or the
+     * default @ref GeneratorProfile.
```

Comment:
The `generatorProfile` is always there, so can delete the "if present..." here and below.

Reply:
b4c2d2e6